### DISCUSSION TITLE
[Feature Request] Export and import setting + files

### DISCUSSION THREAD

**Discussion by cuongthachHQ** [wind-press/windpress#10] [General]
I hope to be able to export and import my settings and files, such as CSS and JS that I have added.

I can't keep recreating new sites or copying and pasting for each client.

Clients should just be able to import a zip file, and voilà, it’s done.

Please add this feature, as it is important for me to create site templates.

![image](https://github.com/user-attachments/assets/2b31948d-3c94-4fee-a88e-12d8d82f8492)

**Comment by cuongthachHQ** [comment]
@suabahasa

**Reply by suasgn** [reply]
Hi, I would appreciate it if you didn't tag/mention as it will double the notification email in the inbox

**Reply by cuongthachHQ** [reply]
Sorry,  I’ll make sure not to tag or mention you in the future.

**Comment by nguyenquanghoang** [comment]
[Manage Option including Import/Export and Reset Options](https://r.freemius.com/5115/3489984/https://creativethemes.com/blocksy/docs/general-options/manage-options/) is a feature I really love in [Blocksy](https://r.freemius.com/5115/3489984/https://creativethemes.com/blocksy/docs/general-options/manage-options/). Hopefully it will also be available in WindPress, very useful for you as an Agency or an individual developing many site.

**Comment by suasgn** [comment]
Added this feature in version 3.2.27 (#19)
